src/main/java/org/infinispan/client/rest/RestCacheManager.java

```diff
@@ -19,13 +21,12 @@
 
    private String host;

```

Comment:
Since in the future we will be connecting to multiple host:port pairs, create a Server class which encapsulates this  information.